lib/request.js

```diff
@@ -302,6 +302,13 @@ Request.prototype.getVersion = function getVersion() {
 };
 Request.prototype.version = Request.prototype.getVersion;
 
+Request.prototype.matchedVersion = function matchedVersion() {
+    if (this._matchedVersion !== undefined) {
```

Comment:
Sure, I didn't change any of the code that came from @nathanpeck, just added tests.